testing/tiptop/APKBUILD

```diff
@@ -0,0 +1,28 @@
+#-*-mode: Shell-script; coding: utf-8;-*-
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+pkgname=tiptop
+pkgver=2.3
+pkgrel=0
+pkgdesc="Tiptop: Hardware Performance Counters for the Masses"
+url="http://tiptop.gforge.inria.fr"
+arch="x86_64"
```

Comment:
It builds on almost any platform, but not every platform has implemented the requisite perf counters. I'm only aware of these existing for x86_64 and i386 on intel cpus. Power cpus might have an implementation as well I've no idea, arm however I know lacks the syscall and requisite hardware support.

I'm not sure what the appropriate call is here, this binary really only works on architectures that support the sys call and implement the requisite hardware to support the sys call in the perf framework. I'll add i386 to the arches as I'm sure that one works too based off the intel references, but I'm not sure having a binary that doesn't work is too useful elsewhere. Thoughts?

Example run on an armhf system:
```
$ tiptop
syscall: Not supported
Could not perform syscall.
Don't know why...
# from strace...
open("/proc/sys/kernel/perf_counter_paranoid", O_RDONLY|O_LARGEFILE) = -1 ENOENT (No such file or directory)
open("/proc/sys/kernel/perf_event_paranoid", O_RDONLY|O_LARGEFILE) = 3
readv(3, [{iov_base="", iov_len=0}, {iov_base="2\n", iov_len=1024}], 2) = 2
_llseek(3, -1, [1], SEEK_CUR)           = 0
close(3)                                = 0
perf_event_open({type=PERF_TYPE_HARDWARE, size=0 /* PERF_ATTR_SIZE_??? */, config=PERF_COUNT_HW_CPU_CYCLES, ...}, 0, -1, -1, 0) = -1 EOPNOTSUPP (Not supported)
```